app/app.go

```diff
@@ -662,6 +662,7 @@ func initModuleAccountPermissions() map[string][]string {
 		axelarnetTypes.ModuleName:      {authtypes.Minter, authtypes.Burner},
 		rewardTypes.ModuleName:         {authtypes.Minter},
 		wasm.ModuleName:                {authtypes.Burner},
+		nexusTypes.ModuleName:          nil,
```

Comment:
this will blacklist nexus module account from sending tokens, do we depend on that anywhere (i don't recall)?